internal/api/registry.go

```diff
@@ -23,13 +23,13 @@ import (
 )
 
 const (
-	ProviderNamespace               = "Microsoft.RedHatOpenShift"
-	ProviderNamespaceDisplay        = "Azure Red Hat OpenShift"
-	ClusterResourceTypeName         = "hcpOpenShiftClusters"
-	NodePoolResourceTypeName        = "nodePools"
-	OperationResultResourceTypeName = "hcpOperationResults"
-	OperationStatusResourceTypeName = "hcpOperationsStatus"
-	ResourceTypeDisplay             = "Hosted Control Plane (HCP) OpenShift Clusters"
+	ProviderNamespace                 = "Microsoft.RedHatOpenShift"
+	ProviderNamespaceDisplay          = "Azure Red Hat OpenShift"
+	ClusterResourceTypeName           = "hcpOpenShiftClusters"
+	NodePoolResourceTypeName          = "nodePools"
+	OperationResultResourceTypeName   = "hcpOperationResults"
+	OperationStatusesResourceTypeName = "hcpOperationsStatuses"
```

Comment:
I think we agreed over Slack to make it `hcpOperationStatuses` (double plurals is weird).

Also I'd prefer if the constant name stays singular like the others.  It reads better in code (imo).